io.catenax.single_level_bom_as_specified/2.0.0/SingleLevelBomAsSpecified.ttl

```diff
@@ -0,0 +1,211 @@
+  samm:optional "true"^^xsd:boolean
+] [
+  samm:property :lastModifiedOn;
+  samm:optional "true"^^xsd:boolean
+] [
+  samm:property :createdOn;
+  samm:optional "true"^^xsd:boolean
+] [
+  samm:property :partGeometry;
+  samm:optional "true"^^xsd:boolean
+]).
+:partVersion a samm:Property;
+    samm:preferredName "Part Version"@en;
+    samm:description "This is the version of the part. The engineering will at times supercede an older part version by a newer one, which might have different material aspects, physical dimensions etc., still maintaining compatibility."@en;
+    samm:characteristic samm-c:Text;
+    samm:exampleValue "05".
+:partQuantity a samm:Property;
+    samm:preferredName "Part Quantity"@en;
+    samm:description "This is the quantity how often this part is in the item."@en;
+    samm:characteristic :PartQuantityCharacteristic.
+:partDescription a samm:Property;
+    samm:preferredName "Part Description"@en;
+    samm:description "The description of the part in the primary language of the production facility of the product owner."@en;
+    samm:characteristic samm-c:Text;
+    samm:exampleValue "The steering wheel is nice and round".
+:partClassification a samm:Property;
+    samm:preferredName "Part Classification"@en;
+    samm:description "The part classification."@en;
+    samm:characteristic :PartClassificationCharacteristic.
+:lastModifiedOn a samm:Property;
+    samm:preferredName "Last Modified On"@en;
+    samm:description "The time the item was modified the last time"@en;
+    samm:characteristic samm-c:Timestamp;
+    samm:exampleValue "2022-02-03T14:48:54.709Z"^^xsd:dateTime.
+:createdOn a samm:Property;
+    samm:preferredName "Created On"@en;
+    samm:description "The time the item was created on"@en;
+    samm:characteristic samm-c:Timestamp;
+    samm:exampleValue "2022-02-03T14:48:54.709Z"^^xsd:dateTime.
+:partGeometry a samm:Property;
+    samm:preferredName "Part Geometry"@en;
+    samm:description "Some parts are intended to be used on either the left or right side, e.g. mirrors, hence this entity describes on which side a part is to be used"@en;
+    samm:characteristic :PartGeometryCharacteristic;
+    samm:exampleValue "left".
+:PartQuantityCharacteristic a samm:Characteristic;
+    samm:preferredName "Part Quantity Characteristic"@en;
+    samm:description "The characteristic of the part quantity."@en;
+    samm:dataType :PartQuantityEntity.
+:PartQuantityEntity a samm:Entity;
+    samm:preferredName "Part Quantity Entity"@en;
+    samm:description "The entity encapsulating the properties describing the quantity of a part."@en;
+    samm:properties (:measurementUnit :quantityNumber :measurementUnit :quantityNumber).
+:measurementUnit a samm:Property;
+    samm:preferredName "Measurement Unit"@en;
+    samm:description "Unit of measurement for the quantity of serialized objects."@en;
+    samm:characteristic samm-c:UnitReference;
+    samm:exampleValue "kW"^^samm:curie.
```

Comment:
this currently produces a validation error as it is not matching the provided regex.

.childParts[0].part[0].partQuantity.measurementUnit | should match pattern "[a-zA-Z]*:[a-zA-Z]+" | "kW"
-- | -- | --

However, kW seems pretty obviously be a correct measurement unit so I am currently investigating, what the problem could be.

Reply:
resolved